includes/Settings.php

```diff
@@ -117,6 +119,22 @@ class Settings extends Service_Base {
 	 */
 	public const SETTING_NAME_SHOPIFY_ACCESS_TOKEN = 'web_stories_shopify_access_token';
 
+	/**
+	 * Shopping_Vendors instance.
+	 *
+	 * @var Shopping_Vendors Shopping_Vendors instance.
+	 */
+	private $shopping_vendors;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Shopping_Vendors $shopping_vendors Shopping_Vendors instance.
+	 */
+	public function __construct( Shopping_Vendors $shopping_vendors ) {
+		$this->shopping_vendors = $shopping_vendors;
+	}
```

Comment:
@swissspidy Instead of injecting this, should do inject the other way around and just register the setting there.

Reply:
_All_ our existing settings are centrally registered in the `Settings` class, so this would break that pattern. Something for a follow-up ticket to look into though.